featurebyte/service/working_schema.py

```diff
@@ -77,30 +78,37 @@ def __init__(self, user: Any, persistent: Persistent, celery: Celery, catalog_id
                 view_construction_service=self.view_construction_service
             ),
         )
-        self.task_manager = TaskManager(
+        task_manager = TaskManager(
             user=user, persistent=persistent, celery=celery, catalog_id=catalog_id
         )
-        self.online_store_table_version_service = OnlineStoreTableVersionService(
+        online_store_table_version_service = OnlineStoreTableVersionService(
             user=user, persistent=persistent, catalog_id=catalog_id
         )
-        self.tile_scheduler_service = TileSchedulerService(
+        tile_scheduler_service = TileSchedulerService(
```

Comment:
if you want to review https://github.com/featurebyte/featurebyte/pull/1462 - this should alleviate these~